packages/api/cms-api/src/generator/generate.ts

```diff
@@ -43,20 +43,3 @@ const program = new Command();
 program.addCommand(generate);
 
 program.parse();
-
-async function getORM(warnWhenNoEntities?: boolean, opts: Partial<Options> = {}): Promise<MikroORM> {
```

Comment:
I didn't get this to work. So I've added the CLI package as peer dependency instead and used their helper, which does the same.